bin/build-release

```diff
@@ -36,6 +36,7 @@ ME=$(readlink -f "$0")
 MY_D=${ME%/*}
 PATH=${MY_D}:$PATH
 QUIET=${QUIET:-}
+BOOTTEST=${BOOTTEST:-}
 
```

Comment:
lets make this be "true" or "false", default to "false" if you want.

    BOOTTEST=${BOOTTEST:-"false"}
    case "$BOOTTEST" in
        true|false) :;;
        *) echo "Unknown value for BOOTTEST: ${BOOTTEST}"; exit 1;;
      esac
    esac